package.json

```diff
@@ -6,24 +6,22 @@
   "main": "main.js",
   "types": "main.d.ts",
   "napi": {
-    "name": "scylladb-javascript-driver",
-    "triples": {
-      "additional": [
-        "aarch64-apple-darwin",
-        "aarch64-linux-android",
-        "aarch64-unknown-linux-gnu",
-        "aarch64-unknown-linux-musl",
-        "aarch64-pc-windows-msvc",
-        "armv7-unknown-linux-gnueabihf",
-        "armv7-unknown-linux-musleabihf",
-        "x86_64-unknown-linux-musl",
-        "x86_64-unknown-freebsd",
-        "i686-pc-windows-msvc",
-        "armv7-linux-androideabi",
-        "universal-apple-darwin",
-        "riscv64gc-unknown-linux-gnu"
-      ]
-    }
+    "binary-name": "scylladb-javascript-driver",
```

Comment:
🌱 We should probably rename `scylladb-javascript-driver` to `scylladb-nodejs-rs-driver`.